webapp/publisher/snaps/views.py

```diff
@@ -499,6 +499,37 @@ def post_release(snap_name):
     return flask.jsonify(response)
 
 
+@publisher_snaps.route("/<snap_name>/release/close-channel", methods=["POST"])
+@login_required
+def post_close_channel(snap_name):
+    data = flask.request.json
+
+    if not data:
+        return flask.jsonify({})
+
+    if data["id"]:
+        snap_id = data["id"]
+    else:
+        return flask.jsonify({})
+
+    if data["info"]:
+        json = "info"
+    else:
+        return flask.jsonify({})
+
+    try:
+        response = api.post_close_channel(flask.session, snap_id, json)
+    except ApiResponseErrorList as api_response_error_list:
+        if api_response_error_list.status_code == 404:
+            return flask.abort(404, "No snap named {}".format(snap_name))
+        else:
+            return flask.jsonify(api_response_error_list.errors), 400
+    except ApiError as api_error:
+        return _handle_errors(api_error)
+
+    return flask.jsonify(response)
```

Comment:
It seems that the success response from API doesn't have `success: true` in data (in similar way that release API does). It would be very helpful if we could add that on our side for successful responses, so they are compatible with release API.

Reply:
ok I can add this field. But we should be careful, this is something we should request to the snap store team instead, especially if we proxy the API calls like this.

Reply:
Yes, makes sense to propose it to store API. Especially that release endpoint has that.

Still would be good to add it ourselves before it's added in the store. Thanks!